client/landing/gutenboarding/onboarding-block/loading-placeholder/index.tsx

```diff
@@ -0,0 +1,43 @@
+/**
+ * External dependencies
+ */
+import React from 'react';
+
+/**
+ * Internal dependencies
+ */
+import Spinner from 'components/spinner';
+import './style.scss';
+
+/* eslint-disable wpcalypso/jsx-classname-namespace */
+export default function LoadingPlaceholder() {
+	return (
+		<div className="editor__placeholder">
+			<div className="edit-post-layout">
```

Comment:
The `edit-post-*` `<div />`s in here seem to recreate Gutenberg's component hierarchy (for styling purposes, I assume). That feels kinda off _inside_ of a block. Is this needed in order to make the placeholder fullscreen-ish? Is there any way we can limit the class names in here to our block's scope?

(Going forward, I'd hope to make Gutenboarding completely `edit-post`-agnostic. After all, `edit-post` is the Gutenberg part that's specifically dedicated to post editing -- and here, we only want a slimmed down, headless editor.)

Reply:
> Is this needed in order to make the placeholder fullscreen-ish?

They aren't needed to make it fullscreen, but they are needed to put the blinking placeholder bars in their respective positions according to where they would appear in the standard editor.

> Is there any way we can limit the class names in here to our block's scope?

By this I assume you mean updating them from `.edit-post-header` to something like `.create-site__header` ?  If we did this the entire style and position of the placeholders will break down, requiring us to re-write all of the css that has already been defined for these classes.  This does not seem like a great use of time for a temporary loading style.

Once we have another editor for styles to mimic, these `.edit-post-*` classnames and placeholder structure could be updated to ensure the style is matching the new editor instead.  But that is assuming that design will decide to keep this as an editor loading screen and not some other 'creating your site' style design.

> Going forward, I'd hope to make Gutenboarding completely edit-post-agnostic. After all, edit-post is the Gutenberg part that's specifically dedicated to post editing

That definitely makes sense to me, but using `.edit-post-*` to piggy-back some pre-defined styles for the purpose of an isolated and temporary loading screen seems acceptable.

Reply:
Note - ignore the "outdated" flag, the directory was renamed to `create-site` which triggered that.

Reply:
I updated the new file under `create-site` to get rid of the `edit-post-*`s.